cucoslib/workers/recommender.py

```diff
@@ -431,11 +456,134 @@ def find_relative_similarity(self, input_stack, input_stack_vectors, filtered_re
+        """Calls the Janus model with the user profile information
+            to get the user_persona"""
+        _janus_end_point = "http://janus-demo-janus.dev.rdu2c.fabric8.io/api/v1/janus"
+        try:
+            if user_profile is not None:
+                response = get_session_retry().post(_janus_end_point, json=user_profile)
+                if response.status_code != 200:
+                    _logger.error ("HTTP error {}. Error retrieving Janus data.".format(response.status_code))
+                    return None
+                else:
+                    json_response = response.json()
+                    return json_response
+            else:
+                _logger.info('User Profile information is not passed in the call, Quitting! Janus model\'s call')
+        except:
+            _logger.error ("Failed retrieving Janus data.")
+            return None
+
+    def call_pgm(self, payload):
+        """Calls the PGM model with the normalized manifest information to get the relevant packages"""
+        _pgm_end_point = "http://kronos-kattappa-0237.dev.rdu2c.fabric8.io/api/v1/kronos_score"
+        try:
+            if payload is not None:
+                print(_pgm_end_point)
+                print(type(payload))
+                response = get_session_retry().post(_pgm_end_point, json=payload)
+                if response.status_code != 200:
+                    _logger.error ("HTTP error {}. Error retrieving PGM data.".format(response.status_code))
+                    return None
+                else:
+                    json_response = response.json()
+                    return json_response
+            else:
+                _logger.info('Payload information is not passed in the call, Quitting! PGM\'s call')
+        except:
+            _logger.error ("Failed retrieving PGM data.")
+            return None
+        
 
     def execute(self, arguments=None):
         arguments = self.parent_task_result('GraphAggregatorTask')
         rs = RelativeSimilarity()
 
+        results = arguments.get('result')
+        user_profile = arguments.get('user_profile')
+
+        # Send the user_profile information to Janus model to get back the user_persona
+        # Which is then used as an input along with a few others to hit PGM 
+        # Which gives us the response with list of packages and that is later on passed to Gremlin to get the version information for those packages
+        
+        janus_response = self.call_janus_model(user_profile)
+        inputTaskForPGM = []
+
+        for result in results:
+            details = result['details'][0]
+            resolved = details['_resolved']
+            self.log.info(result)
+            newArr = [r['package'] for r in resolved]
+            jsonObject = {
+                'ecosystem': details['ecosystem'],
+                'user_persona': "1",
+                'package_list': newArr
+            }
+            self.log.info(jsonObject)
```

Comment:
If it's a debug left-over, then either remove it or turn it into `self.log.debug(json_object)`